src/pipe.ts

```diff
@@ -1,95 +1,95 @@
-import { Operator, Source, Sink, Callbag } from './types';
+import { Operator, Source, Sink } from './types';
 
 // Returning a new callbag
 
 // prettier-ignore
-export function pipe<A, B>(source: Source<A> | Callbag<A>, o1: Operator<A, B>): Callbag<B>;
```

Comment:
Removing the Callbag here was also done in your PR, why? I added it so `subject` could be used in `pipe`.

Reply:
Oh, I didn't realize that `subject` could be in `pipe`, good point!